server/webapp/WEB-INF/rails.new/app/views/admin/stages/index.html.erb

```diff
@@ -10,17 +10,15 @@
             <% if @template_list.length > 0 %>
                 <div class="fieldset">
                     <div class="form_item">
-
                         <div class="template_selection">
-                            <%= f.label l.string("SELECT_TEMPLATE") %>
-                            <%= f.select(com.thoughtworks.go.config.PipelineConfig::TEMPLATE_NAME, @template_list, {:selected => @pipeline.getTemplateName()}, {:id => "select_template"}) %>
-
-                            <% template_name = (@pipeline.getTemplateName() || @template_list.first).to_s %>
+                          <%= f.label l.string("SELECT_TEMPLATE") %>
+                          <%= f.select(com.thoughtworks.go.config.PipelineConfig::TEMPLATE_NAME, @template_list, {:selected => @pipeline.getTemplateName()}, {:id => "select_template"}) %>
 
-                            <%= link_to l.string("VIEW"), "#", :class=> "view_template_link action_icon button preview_icon skip_dirty_stop"%>
-                            <% if is_user_an_admin? %>
-                                <%=link_to l.string("EDIT"), template_edit_path(:pipeline_name => template_name, :stage_parent => "templates", :current_tab => 'general'), :class=> "action_icon edit_icon edit_template_link"%>
-                            <% end %>
+                          <% template_name = (@pipeline.getTemplateName() || @template_list.first).to_s %>
+                              <%= link_to l.string("VIEW"), "#", :class => "view_template_link action_icon button preview_icon skip_dirty_stop" %>
```

Comment:
view link should be added/enabled based on user's permissions. If the allowgroup-admin flag on template is turned off, this link shouldn't be available